src/main/java/edu/harvard/iq/dataverse/harvest/server/web/servlet/OAIServlet.java

```diff
@@ -193,10 +193,48 @@ private RepositoryConfiguration createRepositoryConfiguration() {
                 .withMaxListRecords(100)
                 .withMaxListSets(100)
                 .withEarliestDate(new Date());
-        
+        String descriptionXml = getDescriptionXml();
+        if (descriptionXml != null) {
+            repositoryConfiguration.withDescription(descriptionXml);
+        }
         return repositoryConfiguration; 
     }
-    
+
+    private String getDescriptionXml() {
+        try {
+            // Builds the XML
+            Document doc = DocumentBuilderFactory.newInstance().newDocumentBuilder().newDocument();
+            Element oaiIdentifier = doc.createElement("oai-identifier");
+            oaiIdentifier.setAttribute("xmlns", "http://www.openarchives.org/OAI/2.0/oai-identifier");
+            oaiIdentifier.setAttribute("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance");
+            oaiIdentifier.setAttribute("xsi:schemaLocation", "http://www.openarchives.org/OAI/2.0/oai-identifier http://www.openarchives.org/OAI/2.0/oai-identifier.xsd");
+            doc.appendChild(oaiIdentifier);
+            Element scheme = doc.createElement("scheme");
+            scheme.appendChild(doc.createTextNode("oai"));
+            oaiIdentifier.appendChild(scheme);
+            Element repositoryIdentifier = doc.createElement("repositoryIdentifier");
+            repositoryIdentifier.appendChild(doc.createTextNode("dataverse.org"));
```

Comment:
In #4597 @JingMa87 and I have been discussing making "repositoryIdentifier" (and possibly other identifiers configurable).